apps/studio/src/features/editing-experience/components/ComplexEditorStateDrawer.tsx

```diff
@@ -269,39 +263,39 @@ export default function ComplexEditorStateDrawer(): JSX.Element {
           </HStack>
         </Box>
 
-        <Box px="2rem" py="1rem">
+        {/* NOTE: reserve at least 4.75rem for the bottom bar */}
```

Comment:
how did we get 4.75rem? feels like a magic number

Reply:
this was gotten by observing the save button - it had 44height, and 24px top and bottom padding, adding to 92 px or 4.75rem